upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -149,6 +149,22 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		}
 	}
 
+	if cluster.Spec.Networking != nil && cluster.Spec.Networking.AmazonVPC != nil {
```

Comment:
Use `WithDefaultBool` in the template.

Reply:
Removed the template functions and switched to use `WithDefaultBool` as suggested. Thanks!